### PR TITLE
added percentage display to stats

### PR DESCRIPTION
**Pull request by athompsonScottLogic**

### CODE REVIEW

File: whichone/static/js/ui.js [hunk 1]
```diff
@@ -143,9 +145,9 @@ function getStats(param1, param2) {
     plural = "s";
 
     if (currentMode == 'danceability') {
-        document.getElementById("stats-text").textContent = bigChoice + " is " + timesMore + " more danceable than " + smallChoice + ".";
+        document.getElementById("stats-text").textContent = bigChoice + " is " + percentMore + " more danceable than " + smallChoice + ".";
     } else if (currentMode == 'valence') {
-        document.getElementById("stats-text").textContent = bigChoice + " is " + timesMore + " more upbeat than " + smallChoice + ".";
+        document.getElementById("stats-text").textContent = bigChoice + " is " + percentMore + " more upbeat than " + smallChoice + ".";
```
Comment (connor-settle) [inline whichone/static/js/ui.js]: Could use [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

**Review by connor-settle — APPROVED** [review]
Looks good

File: whichone/static/js/ui.js (hunk 1 above)
Comment (athompsonScottLogic) [reply whichone/static/js/ui.js]: Converted to interpolate strings

**Review by mcgill-a — APPROVED** [review]
nice refactor

**Pull request merged**